Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: push-image
+push-image:
```

Comment:
Not too sure if push-image is the right name. I think it might only build right now

Reply:
@simonswine I know it's confusing, but `docker buildx build -o type=registry` pushes to the repository. I think it's the currently sanctioned way of pushing manifest lists.